src/cluster/websocket/WebSocket.cc

```diff
@@ -303,7 +303,6 @@ void WebSocketEventDispatcher::Process(const WebSocketOpen& open) {
     });
 
     if ( ! good_application_name ) {
```

Comment:
Just FTR, like discussed offline, I don't think `good_application_name` is ideal since it leaks Prometheus label restrictions into something only tangentially related (the names users want to give their websocket clients), and I'd tend more towards just sanitizing the name once it gets exported as a metric (but that could lead to mismatches between app names and metric labels).

If we keep it we should document `X-Application-Name` and its rules in the WebSocket API docs, probably as part of https://github.com/zeek/zeek-docs/issues/331.